src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -706,6 +761,18 @@ public void PlaceOrganelle()
         }
     }
 
+    public void ResetCamera()
+    {
+        camera.CameraHeight = camera.DefaultCameraHeight;
+        camera.Translation = new Vector3(0, camera.Translation.y, 0);
```

Comment:
I think these lines should be put in the camera class, make a new method there called ResetPosition. Or rather should this reset the position to 0,0,0 for the looked at object? That would be much cleaner, I think.